src/style.css

```
@@ -0,0 +1,122 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-size: 11px;
+  margin: 0;
+}
+
+p,
+h3 {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  justify-content: space-between;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 30px 14px 14px;
+}
+
+.card__image {
+  display: flex;
+  justify-content: center;
+  
+}
+
+.card__picture {
+  height: 134px;
+  width: 160px;
+}
+
+.card__title {
+  font-size: 12px;
+  line-height: 15px;
+  margin-top: 45px;
+  font-weight: 500;
+}
+
+.card__cod {
+  width: 100%;
+  text-align: left;
+  line-height: 13px;
+  margin-top: 5px;
+}
+
+.card__rating {
+  display: flex;
+  width: 100%;
+  justify-content: space-between ;
+  padding-top: 15px;
+}
+
+.card__stars {
+  display: flex;
+}
+
+.card__star {
+  background: url("images/star.svg") no-repeat center;
+  height: 15px;
+  width: 15px;
+  margin-right: 4px;
+}
+
+.card__stars--stars--1 :nth-child(1),
+.card__stars--stars--2 :nth-child(-n+2),
+.card__stars--stars--3 :nth-child(-n+3),
+.card__stars--stars--4 :nth-child(-n+4),
+.card__stars--stars--5 :nth-child(-n+5) {
+  background: url("images/star-active.svg") no-repeat center;
```

Ці параметри можна не повторювати.
```suggestion
  background: url("images/star-active.svg");
```